ext/mysqli/tests/deprecated_constants.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+Deprecated constants
+--SKIPIF--
+<?php
+require_once('skipif.inc');
+?>
+--FILE--
+<?php
+
+echo constant('MYSQLI_NO_DATA')."\n";
+echo constant('MYSQLI_DATA_TRUNCATED')."\n";
+echo constant('MYSQLI_SERVER_QUERY_NO_GOOD_INDEX_USED')."\n";
+echo constant('MYSQLI_SERVER_QUERY_NO_INDEX_USED')."\n";
+echo constant('MYSQLI_SERVER_QUERY_WAS_SLOW')."\n";
+echo constant('MYSQLI_SERVER_PS_OUT_PARAMS')."\n";
+
+print "done!";
```

Comment:
No need for the final done.

Reply:
What's the point of Done in test cases? I removed them from both, is this correct?

Reply:
We commonly add it for tests that would be completely empty otherwise, but I don't think it's strictly necessary. If the test crashes, a message is inserted.

Reply:
That used to help to detect crashes.

Reply:
Now that the crash detection is also implemented on Windows, there's no reason to do this anymore, right?

Reply:
Right, *used* to help. :)